src/components/molecules/search/Search.jsx

```diff
@@ -40,31 +58,91 @@ const Search = () => {
-                <Radio label={i18n.contains[language]}
+                <Radio label={i18n.containsRadioLabel[language]}
                        id="contains"
                        className="inline"
-                       value={containsValue}
+                       value={searchMatchType.contains}
+                       defaultChecked={selectedOption === searchMatchType.contains}
                        onChange={toggleRadioSelection} />
             </div>
-            <TextInput id="keywords"
-                       action={updateTextInput}
-                       classes="dictionary-search-input"
-                       label=""
-                       placeHolder={i18n.searchPlaceholderText[language]} />
+
+            <Autocomplete
+                id="keywords"
+                label={ i18n.searchPlaceholderText[language] }
+                labelHidden
+                wrapperClasses="dictionary-search-input"
+                inputClasses="dictionary-search-input"
+                value={ searchText }
+                inputProps={{
+                    placeholder: i18n.searchPlaceholderText[language]
+                }}
+                items={ autoSuggest.payload || [] }
+                getItemValue={ item => item.termName }
+                shouldItemRender={ matchItemToTerm }
+                onChange={ event => onChangeHandler( event ) }
+                onSelect={ ( value, item ) => onSelectHandler( value ) }
+                renderMenu={ children => (
+                    <div
+                        className="ncids-autocomplete__menu --terms"
+                        data-testid={testIds.AUTO_SUGGEST_OPTIONS}
+                    >
+                        { searchText.length >= 3 && autoSuggest.payload
+                            ? autoSuggest.payload.length
+                                ? children
+                                : (
+                                    /* Todo: Get spanish language text translation */
+                                    <div className="ncids-autocomplete__menu-item">
+                                        {i18n.autosuggestNoResultsFound[language]}
+                                    </div>
+                                )
+                            : (
+                                /* Todo: Get spanish language text translation */
+                                <div className="ncids-autocomplete__menu-item">
+                                    {i18n.autoSuggestThreeOrMoreChars[language]}
+                                </div>
+                            )
+                        }
+                    </div>
+                ) }
+                renderItem={ ( item, isHighlighted ) => (
+                    <div
+                        className={`ncids-autocomplete__menu-item ${
+                            isHighlighted ? 'highlighted' : ''
+                        }`}
+                        key={item.termId}
```

Comment:
Could you add the attribute `role="option"` here?

Reply:
Sure, will do.

Reply:
Not as a child of `listbox`.  Children of a listbox should be `option`

Reply:
Will update.

Reply:
Updated.